dotCMS/src/main/java/com/dotcms/util/content/json/PopulateContentletAsJSONUtil.java

```diff
@@ -0,0 +1,398 @@
+                PGobject jsonObject = new PGobject();
+                jsonObject.setType("json");
+                Try.run(() -> jsonObject.setValue(line)).getOrElseThrow(
+                        () -> new IllegalArgumentException("Invalid JSON"));
+                contentletAsJSON = jsonObject;
+            } else {
+                contentletAsJSON = line;
+            }
+
+            this.paramsUpdate.add(new Params(contentletAsJSON, contentlet.inode()));
+
+            // Execute the batch for the updates if we have reached the max batch size
+            if (this.paramsUpdate.size() >= MAX_UPDATE_BATCH_SIZE) {
+                this.doUpdateBatch(totalInsertAffected);
+            }
+        } catch (JsonProcessingException e) {
+            throw new DotRuntimeException("Error processing line", e);
+        }
+    }
+
+    /**
+     * Converts the contentlet to an immutable contentlet and then builds a json representation of it.
+     *
+     * @param contentlet
+     * @return The Contentlet with the json representation attached to it.
+     */
+    private String toJSON(Contentlet contentlet) {
+
+        try {
+            // Converts the given contentlet to an immutable contentlet and then builds a json representation of it.
+            var contentletAsJSON = ContentletJsonHelper.INSTANCE.get().writeAsString(this.contentletJsonAPI.toImmutable(contentlet));
+
+            // I need to have the JSON in a single line, so I can write it into a file
+            return contentletAsJSON.replaceAll("[\\t\\n\\r]", "");
+        } catch (JsonProcessingException e) {
+            throw new DotRuntimeException("Error creating the JSON representation of the Contentlet", e);
+        }
+    }
+
+    /**
+     * Executes the batch of updates to fill the contentlet_as_json column.
+     *
+     * @param totalUpdateAffected
+     */
+    private void doUpdateBatch(final MutableInt totalUpdateAffected) {
+
+        try {
+            final List<Integer> batchResult =
+                    Ints.asList(new DotConnect().executeBatch(
+                            UPDATE_CONTENTLET_AS_JSON,
+                            this.paramsUpdate));
+
+            final int rowsAffected = batchResult.stream().reduce(0, Integer::sum);
+            Logger.info(this, "Batch rows to populate contentlet_as_json column, updated: " + rowsAffected + " rows");
+            totalUpdateAffected.add(rowsAffected);
+        } catch (DotDataException e) {
+            Logger.error(this, "Couldn't update these rows: " + this.paramsUpdate);
+            Logger.error(this, e.getMessage(), e);
+        } finally {
+            this.paramsUpdate.clear();
+        }
+    }
+
+    /**
+     * This basically tells Weather or not we support saving content as json and if we have not turned it off.
+     */
+    public static boolean canPersistContentAsJson() {
+        return isJsonSupportedDatabase()
+                && Config.getBooleanProperty(SAVE_CONTENTLET_AS_JSON, true);
```

Comment:
we have found the Config is a blocking class, we used to do things such as

private static final Lazy<Boolean> SAVE_CONTENTLET_AS_JSON = Lazy.of (()->Config.getBooleanProperty(SAVE_CONTENTLET_AS_JSON, true));

To avoid using the config every time